.changeset/silly-geckos-sip.md

```diff
@@ -0,0 +1,9 @@
+---
+'reassure': minor
```

Comment:
It should be a patch update, as there is no visible change to the user here.

Reply:
Thanks, done @mdjastrzebski